lib/cadet/auth/error_handler.ex

```diff
@@ -2,13 +2,18 @@ defmodule Cadet.Auth.ErrorHandler do
   @moduledoc """
   Handles authentication errors
   """
+  use CadetWeb, :controller
   import Plug.Conn
 
   def auth_error(conn, {type, _reason}, _opts) do
-    body = to_string(type)
+    if type == :unauthenticated do
```

Comment:
You don't need to handle the other case for now (which is `:already_authenticated`). It is used by `Guardian.Plug.EnsureNotAuthenticated`

Reply:
So simply redirect, you can assume that `type` is always `:unauthenticated`

Reply:
Also the error code should still be 401